packages/config/config/devices/0x0345/templates/swidget_template.json

```
@@ -0,0 +1,1504 @@
+	},
+	"humidity_low_actions_control_host_on_entry": {
+		// param 143b2
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Humidity Low Actions: CONTROL HOST ON ENTRY",
+		"description": "If set, execute BASIC SET with 'Humidity Low Entry SET Value' on host device on low/under humidity state entry"
+	},
+	"humidity_low_actions_control_host_on_exit": {
+		// param 143b3
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Humidity Low Actions: CONTROL HOST ON EXIT",
+		"description": "If set, execute BASIC SET with 'Humidity Low Exit SET Value' on host device on low/under humidity state exit."
+	},
+	"humidity_low_actions_report_transitions": {
+		// param 143b4
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Humidity Low Actions: REPORT TRANSITIONS",
+		"description": "If set, send humidity MULTILEVEL SENSOR REPORT to Lifeline on high/over humidity state entry and exit"
+	},
+	"humidity_low_actions_apply_entry_delay": {
+		// param 143b5
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Humidity Low Actions: APPLY ENTRY DELAY",
+		"description": "If set, apply Humidity Low Delay for low/under humidity state entry"
+	},
+	"humidity_low_actions_apply_exit_delay": {
+		// param 143b6
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Humidity Low Actions: APPLY EXIT DELAY",
+		"description": "If set, apply Humidity Low Delay for low/under humidity state exit."
+	},
+	"humidity_low_actions_baseline_mode": {
+		// param 143b7
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Humidity Low Actions: BASELINE MODE",
+		"description": "If set, use humidity averaging for low/over humidity range checks"
+	},
+	"system_status": {
+		// param 152
+		"label": "System Status",
+		"description": "Used to provide status information for system troubleshooting.",
+		"valueSize": 4,
+		"minValue": 0,
+		"maxValue": 4294967295,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": true
+	},
+	"system_led_brightness": {
+		// param 153
+		"label": "System LED Brightness",
+		"description": "Used to set the power output to the system LED to control on brightness level.",
+		"unit": "%",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 100,
+		"unsigned": true,
+		"defaultValue": 100
+	},
+	"led_invert": {
+		// param 154 b0
+		"label": "LED invert",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "LED follows state of host switch",
+				"value": 0
+			},
+			{
+				"label": "LED is inverse of state of host switch",
+				"value": 1
+			}
+		]
+	},
+	"switch_host_control_flags_3_way_mode": {
+		// param 168b0
+		"label": "Switch Host Control Flags: 3-Way Mode",
```

I think just 
```suggestion
		"label": "3-Way Mode",
```
is enough

---

Fixed